packages/babel-cli/src/babel/index.js

```diff
@@ -154,10 +154,14 @@ commander.option(
   "-D, --copy-files",
   "When compiling a directory copy over non-compilable files",
 );
+commander.option(
+  "--copy-dot-files",
+  "When --copy-files is set, dot files are copied as well",
```

Comment:
Super minor nit, but “dotfiles” is generally written as a single word. Can we change the option to `--copy-dotfiles` and `s/dot files/dotfiles/g`?